### ISSUE TITLE
switch to webpack breaks use of dms-js in node.js server

### ISSUE THREAD

**Issue by matthewberryman** [WSDOT-GIS/dms-js#5]
```
/Users/matthew/code/msf-reach/node_modules/dms-conversion/dms.js:1
(function (exports, require, module, __filename, __dirname) { !function(e,t){"object"==typeof exports&&"object"==typeof module?module.exports=t():"function"==typeof define&&define.amd?define([],t):"object"==typeof exports?exports.dms=t():e.dms=t()}(window,function(){return function(e){var t={};function r(n){if(t[n])return t[n].exports;var o=t[n]={i:n,l:!1,exports:{}};return e[n].call(o.exports,o,o.exports,r),o.l=!0,o.exports}return r.m=e,r.c=t,r.d=function(e,t,n){r.o(e,t)||Object.defineProperty(e,t,{configurable:!1,enumerable:!0,get:n})},r.r=function(e){Object.defineProperty(e,"__esModule",{value:!0})},r.n=function(e){var t=e&&e.__esModule?function(){return e.default}:function(){return e};return r.d(t,"a",t),t},r.o=function(e,t){return Object.prototype.hasOwnProperty.call(e,t)},r.p="",r(r.s=0)}([function(e,t,r){"use strict";r.r(t),r.d(t,"Dms",function(){return i}),r.d(t,"parseDms",function(){return s});var n=/^(-?\d+(?:\.\d+)?)[°:d]?\s?(?:(

ReferenceError: window is not defined
    at Object.<anonymous> (/Users/matthew/code/msf-reach/node_modules/dms-conversion/dms.js:1:250)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/matthew/code/msf-reach/dist/lib/tsr.js:15:22)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/matthew/code/msf-reach/dist/api/routes/hazards/index.js:15:12)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! msf-reach@1.1.0 start: `node dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the msf-reach@1.1.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/matthew/.npm/_logs/2018-04-30T01_28_42_163Z-debug.log
```

**Comment by matthewberryman** [comment]
Seems to be caused by https://github.com/webpack/webpack/issues/6642 with temporary fix in place  from webpack 4.4.1 https://github.com/kalcifer/webpack-library-example/pull/8

**Comment by matthewberryman** [comment]
https://github.com/AcrossTheCloud/dms-js/commit/a7f431f4b1736e09fdde134460115bc36d29959e fixes that error, but then my [import](https://github.com/MSFREACH/msf-reach/blob/6fe47d4a62f539abc89e527043e7ddf89c9fcf1f/src/lib/tsr.js#L5) ends up undefined, presumably because of this reason https://github.com/vuejs/vue/issues/7655

**Comment by matthewberryman** [comment]
https://github.com/WSDOT-GIS/dms-js/blob/master/tsconfig.json#L44 defines esModuleInterop as true but then the json is invalid at that point (missing comma) so maybe that's breaking things.

**Comment by matthewberryman** [comment]
correction on my previous comment: the json was valid, I missed spotting that that was the last key/value pair (the rest are commented out) due to spurious syntax highlighting in Atom.

**Comment by matthewberryman** [comment]
In my case, babel is then remapping 'this' to 'undefined': https://babeljs.io/faq#why-is-this-being-remapped-to-undefined
Not sure on an easy solution to this.

**Comment by fkm** [comment]
I reverted back to 3.0.0

**Comment by JeffJacobson** [comment]
I've implemented the changes from #6 and published a "3.1.1-alpha.1" version to npm. Let me know if this resolves the issue.
@matthewberryman @fkm

**Comment by matthewberryman** [comment]
Thanks @JeffJacobson, tested successfully in https://github.com/MSFREACH/msf-reach/commit/92fa65643ef24017a547a6e676712a376504c550
